x-pack/plugins/security_solution/public/overview/components/detection_response/rule_alerts_table/use_rule_alerts_items.test.ts

```diff
@@ -50,16 +50,12 @@ jest.mock('../../../../common/containers/use_global_time', () => {
 
 // helper function to render the hook
 const renderUseRuleAlertsItems = (props: Partial<UseRuleAlertsItemsProps> = {}) =>
-  renderHook<UseRuleAlertsItemsProps, ReturnType<UseRuleAlertsItems>>(
-    () =>
-      useRuleAlertsItems({
-        queryId: 'test',
-        signalIndexName: 'signal-alerts',
-        ...props,
-      }),
-    {
-      wrapper: TestProviders,
```

Comment:
it didnt need TestProviders to render, but let me know if Im missing some intention